lib/themes/dosomething/paraneue_dosomething/js/app.js

```diff
@@ -42,6 +42,11 @@ import 'validators/organDonation';
 
 import 'campaign/prototype';
 
+// Load React components
+import 'components/onboard';
```

Comment:
I think it'd make sense to make a `onboarding.js` entry point. If we end up having other bundles that also use React, it'll get pulled into `lib.js` and shared between bundles automatically.